graph/index.html

```diff
@@ -80,7 +81,7 @@
       <a class="navbar-link">Infrastructure</a>
       <div class="navbar-dropdown">
         <a class="navbar-item">Production</a>
-        <a class="navbar-item">Staging</a>
+        <!-- <a class="navbar-item">Staging</a> -> Removing / unavaialbe -->
```

Comment:
if only production works, lets remove the whole selector

Reply:
Thanks, I needed a second opinion on this, hence commenting it out :) Will do!